contracts/Baal.sol

```diff
@@ -530,12 +532,14 @@ contract Baal is Executor, Initializable {
             uint32 min,
             uint32 max,
             uint32 grace,
+            uint256 newOffering,
```

Comment:
Thinking we should rename this function since setting periods is just one of the things it does